### ISSUE TITLE
merging results of two different strategies is painful and we have no good solution for it

### ISSUE THREAD

**Issue by TimDaub** [neume-network/strategies#236]
- there are cases where two strategies outputs have to be merged into one
- an extreme case of this is the entire implementation of music-os-accumulator
- a more lean example of this problem can be seen in #235 where we need to know from call-block-logs if something is a certain music NFT platform, but it depends on dynamic values becoming apparent only after call-block-logs has run once.

**Comment by TimDaub** [comment]
In addition: music-os-accumulator is strictly dependent on the existence of e.g. a previous strategies files. So e.g. if we invoked call-block-logs with a block range that doesn't include a zora NFT mint, it means that music-os-accumulator probably fails as it won't find any prior files related to the zora crawl, e.g. zora-get-tokenuri. 

With the more frequent block-logs crawl that may not contain any NFT mints for a run (e.g. when we run every ten minutes), we must consider this and appropriately log/error.

Here are two commits that already lessen this problematic:

- https://github.com/neume-network/strategies/commit/aa966a1a76b63c6e6ce76bbd6e8b1b0203daac50
- https://github.com/neume-network/strategies/commit/66814308e36af10bb626877ab715399ede31b3ce

/cc @il3ven for visibility too.
